azure-cs-aks-managed-identity/README.md

```diff
@@ -0,0 +1,56 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/blob/master/azure-cs-aks-managed-identity/README.md)
+
+# Azure Kubernetes Service (AKS) Cluster using the native Azure Provider
+
+This example deploys an AKS cluster, creates an Azure User Assigned Managed Identity, and sets credentials to manage access to the cluster.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Configure Azure Credentials](https://www.pulumi.com/docs/intro/cloud-providers/azure/setup/)
+
+### Steps
+
+After cloning this repo, from this working directory, run these commands:
```

Comment:
Add a link to these steps --> https://github.com/pulumi/examples#checking-out-a-single-example